docs/dev/table/connect.md

```diff
@@ -1075,6 +1075,88 @@ CREATE TABLE MyUserTable (
+<dependency>
+  <groupId>org.apache.flink</groupId>
+  <artifactId>flink-connector-jdbc{{ site.scala_version_suffix }}</artifactId>
+  <version>{{site.version }}</version>
+</dependency>
+{% endhighlight %}
+
+And must also specify JDBC library, for example, if want to use Mysql library, the following dependency to your project:
+
+{% highlight xml %}
+<dependency>
+    <groupId>mysql</groupId>
+    <artifactId>mysql-connector-java</artifactId>
+    <version>8.0.17</version>
+</dependency>
+{% endhighlight %}
+
+**Library support:** Now, we only support mysql, derby, postgres.
+
+The connector can be defined as follows:
+
+<div data-lang="DDL" markdown="1">
+{% highlight sql %}
+CREATE TABLE MyUserTable (
+  ...
+) WITH (
+  'connector.type' = 'jdbc', -- required: specify this table type is jdbc
+  
+  'connector.url' = 'jdbc:derby:memory:upsert', -- required: JDBC DB url
+  
+  'connector.table' = 'jdbc_table_name',  -- required: jdbc table name
+  
+  'connector.driver' = 'driver', -- optional: jdbc driver, if not set, it will automatically guess based on the URL.
+
+  'connector.username' = 'name', -- optional: jdbc user name and password
+  'connector.password' = 'password',
+  
+  -- scan options, optional, used when reading from table
+  'connector.read.partition.column' = 'column_name', -- optional, name of the column used for partitioning the input.
+  'connector.read.partition.num' = '50', -- optional, the largest value of the last partition.
+  'connector.read.partition.lower-bound' = '500', -- optional, the smallest value of the first partition.
+  'connector.read.fetch-size' = '100', -- optional, the maximum number of partitions that can be used for parallelism in table reading.
+
+  -- lookup options, optional, used in temporary join
+  'connector.lookup.cache.max-rows' = '-1', -- optional, lookup cache max size, over this value, the old data will be eliminated.
+  'connector.lookup.cache.ttl' = '-1', -- optional, lookup cache expire mills, over this time, the old data will expire.
+  'connector.lookup.max-retries' = '3', -- optional, max retry times for jdbc connector.
+
+  -- sink options, optional, used when writing into table
+  'connector.write.flush.max-rows' = '5000', -- optional, flush max size (includes all append, upsert and delete records), 
+                                                -- over this number of records, will flush data.
+  'connector.write.flush.interval' = '0', -- optional, flush interval mills, over this time, asynchronous threads will flush data.
+  'connector.write.max-retries' = '3' -- optional, max retry times for jdbc connector.
+)
+{% endhighlight %}
+</div>
+</div>
+
+**Upsert sink:** If a JDBC table is used as upsert sink, please confirm that the primary key of flink sql is the unique key of underlying database.
+
+**Temporary Join:** When enable cache (cacheMaxSize and cacheExpireMs not equal -1), lookup's performance will be greatly enhanced, but it may not read the latest data.
+
```

Comment:
Could we add a note for writing? As default, the `connector.write.flush.interval` is `0s` and `connector.write.flush.max-rows` is `5000`, which means for low traffic queries, the buffered output rows may not be flushed to database for a long time. So the interval configuration is recommended to set.